app/views/applications/index.html.erb

```diff
@@ -6,8 +6,14 @@
   ]
 } %>
 
-<h1>Applications</h1>
+<%= render "govuk_publishing_components/components/title", {
+  title: "Applications"
+} %>
 
 <%= render partial: "applications_table", locals: { applications: @applications, environments: @environments } %>
 
-<%= link_to 'Create application', new_application_path, class: "btn btn-success" %>
```

Comment:
At some point we should think about whether this button should be all the way at the bottom of the page, but that's probably out of scope for this PR.

Reply:
Agreed, that makes sense to consider as a separate piece of work.